gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -328,13 +329,14 @@ const postChangeUnissuedFacilityPreview = async (req, res) => {
     await api.updateApplication({ dealId, application: applicationUpdate, userToken });
 
     const facilityEndDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/facility-end-date/change`;
+    const bankReviewDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/bank-review-date/change`;
     const aboutFacilityRedirectUri = `/gef/application-details/${dealId}`;
 
-    if (isUsingFacilityEndDate === true) {
-      return res.redirect(facilityEndDateRedirectUri);
+    if (isUsingFacilityEndDate === undefined) {
+      return res.redirect(aboutFacilityRedirectUri);
     }
 
-    return res.redirect(aboutFacilityRedirectUri);
+    return res.redirect(isUsingFacilityEndDate ? facilityEndDateRedirectUri : bankReviewDateRedirectUri);
```

Comment:
have these changes been tested?

Reply:
I missed this comment, do we also have unit tests for these?

Reply:
Yes in `gef-ui\server\controllers\unissued-facilities\index.test.js`